tests/pass/current_dir.rs

```diff
@@ -1,3 +1,4 @@
+// ignore-android: No foreign function support for __errno yet
```

Comment:
What does this have to do with what the test is doing?

Reply:
Same as the `test/pass/concurrency/simple.rs` comment this is relating to the panic raised when actually running the test. I won't blow up your notifications on each instance here, but that's the general theme with the `ignore-android` comments.